static/src/stylesheets/module/commercial/_adverts.scss

```diff
@@ -348,6 +348,10 @@
     &[open] .inline-plus {
         display: none;
     }
+
+    &:not([open]) > *:not(summary) {
```

Comment:
This rule is only for browsers that don't support the element yet, we'll get rid of it when it's safe to. I agree but the `open` attribute is how the HTML element works (try in Chrome).